mslice/cli/_mslice_commands.py

```diff
@@ -172,42 +173,7 @@ def Cut(InputWorkspace, CutAxis=None, IntegrationAxis=None, NormToOne=False):
     app.MAIN_WINDOW.cut_plotter_presenter.update_main_window()
     return cut
 
-def PlotSlice(InputWorkspace, IntensityStart="", IntensityEnd="", Colormap=DEFAULT_CMAP):
-    """
-    Creates mslice standard matplotlib plot of a slice workspace.
-    Keyword Arguments:
-    InputWorkspace -- Workspace to plot. The parameter can be either a python
-                      handle to the workspace OR the workspace name as a string.
-    IntensityStart -- Lower bound of the intensity axis (colorbar)
-    IntensityEnd -- Upper bound of the intensity axis (colorbar)
-    Colormap -- Colormap name as a string. Default is 'viridis'.
-    """
-    _check_workspace_name(InputWorkspace)
-    workspace = get_workspace_handle(InputWorkspace)
-    _check_workspace_type(workspace, HistogramWorkspace)
-    slice_presenter = app.MAIN_WINDOW.slice_plotter_presenter
-    slice_presenter.change_intensity(workspace.name, IntensityStart, IntensityEnd)
-    slice_presenter.change_colourmap(workspace.name, Colormap)
-    slice_presenter.plot_from_cache(workspace)
 
-def PlotCut(InputWorkspace, IntensityStart=0, IntensityEnd=0, PlotOver=False):
-    """
-    Create mslice standard matplotlib plot of a cut workspace.
-    Keyword Arguments:
-    InputWorkspace -- Workspace to cut. The parameter can be either a python
-                      handle to the workspace OR the workspace name as a string.
-
-    IntensityStart -- Lower bound of the y axis
-    IntensityEnd -- Upper bound of the y axis
-    PlotOver -- if true the cut will be plotted on an existing figure.
-    """
-    _check_workspace_name(InputWorkspace)
-    workspace = get_workspace_handle(InputWorkspace)
-    if not isinstance(workspace, HistogramWorkspace):
-        raise RuntimeError("Incorrect workspace type.")
-    if IntensityStart == 0 and IntensityEnd == 0:
-        intensity_range = None
-    else:
-        intensity_range = (IntensityStart, IntensityEnd)
-    app.MAIN_WINDOW.cut_plotter_presenter.plot_cut_from_workspace(workspace, intensity_range=intensity_range,
-                                                                  plot_over=PlotOver)
+def plot(*args, **kwargs):
```

Comment:
We were okay with having the users specify the projection type in their scripts so we shouldn't need this function at all I don't think. I also think the `matplotlib.pyplot` import is what's causing the build failures.

Reply:
The builds fail for the same reason in spite of removing `matplotlib.pyplot`